pywbem/exceptions.py

```diff
@@ -75,16 +80,20 @@ def __str__(self):
 
 
 class ConnectionError(Error):
-    """This exception indicates a problem with the connection to the WBEM
-    server. A retry may or may not succeed. Derived from
-    :exc:`~pywbem.Error`."""
+    """
+    This exception indicates a problem with the connection to the WBEM
+    server. A retry may or may not succeed.
+    Derived from :exc:`~pywbem.Error`.
```

Comment:
Do we really need the derived comment since that is clear from the class definition itself.  No big deal, just trying to remove redundant text.

Reply:
Same comment couple of times below

Reply:
I added that because it is not shown in the documentation and in order to have a link that can be followed.